string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,49 @@
+/* exported titleCase */
+function titleCase(string) {
+  var i;
+
+  // turning string into an array
+  // sorting through dashes and colons
+  var wordArray = [];
+  var word = '';
+  string = string.toLowerCase();
+  for (i = 0; i < string.length; i++) {
+    if (string[i] === '-') {
+      word += string[i];
+      i++;
+      word += string[i].toUpperCase();
+    } else if (string[i] === ':') {
+      word += string[i];
+      wordArray.push(word);
+      word = '';
+      i = i + 2;
+      word += string[i].toUpperCase();
+    } else if (string[i] !== ' ') {
+      word += string[i];
+    } else {
+      wordArray.push(word);
+      word = '';
+    }
+  }
+  wordArray.push(word);
+
+  // sorting through the array
+  // lowercasing words that should be lowercased
+  // uppercasing the first letter of every word + the word JavaScript and API
+  for (i = 0; i < wordArray.length; i++) {
+    if (wordArray[i] === 'javascript') {
+      wordArray[i] = 'JavaScript';
+    } else if (wordArray[i] === 'javascript:') {
```

Comment:
You could use the `includes` method to simplify this 🤓 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes